src/types.ts

```diff
@@ -1,3 +1,5 @@
+import { BigIntArgument } from '@pcd/pcd-types'
```

Comment:
Maybe we should re-export BigIntArgument in types.ts.

Reply:
True!
I'll change it

Reply:
Where do you think it will be useful?

Reply:
Because BigIntArgument in our package and another package use @pcd/pcd-types/BigIntArgument  will be independent.